website/docs/_Node/node.mdx

```diff
@@ -1,6 +1,174 @@
 ---
 id: run-node
-title: Run Node (Coming soon...)
+title: Run a Node
 sidebar_position: 7
 ---
-# Coming soon...
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+Running a CKB node boosts network robustness and decentralization while enhancing data access security.
+
+## Run a CKB Mainnet Node
```

Comment:
Do you think we can break this into two files, one for Mainnet node and the other for public RPC node? 
Also, we might also need guide for running a Testnet node (you can refer to this: https://nervos-ckb-docs-git-v1-cryptape.vercel.app/docs/basics/guides/testnet).

Reply:
How about the following structure?
- Run Node
 --- Run a Mainnet Node
 --- Run a Testnet Node
 --- Run a Public RPC Node